packages/css/src/components/viewer-grid/viewer-grid.stories.ts

```diff
@@ -18,18 +77,43 @@ const main = html`
       himenaeos. Quisque sodales nibh risus, ac dictum diam volutpat quis. Integer sed tortor quis libero iaculis
       interdum.
     </p>
-    <p>
-      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean laoreet ligula vel cursus consequat. Aenean id dolor
-      felis. Mauris eget ullamcorper neque. Donec finibus libero lorem, faucibus sollicitudin dui vehicula eu. Vestibulum
-      libero lorem, rutrum ac blandit euismod, elementum vel diam. Nunc at convallis mi. Aliquam tincidunt ante eu quam
-      molestie, nec rutrum quam accumsan. Pellentesque porta quis sem et dictum. Curabitur varius vel metus non pulvinar.
-      Donec ut magna ut nunc lacinia vestibulum et eget enim. In efficitur felis non massa vulputate, eu pellentesque nisi
-      sagittis. Nulla non magna eros. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos
-      himenaeos. Quisque sodales nibh risus, ac dictum diam volutpat quis. Integer sed tortor quis libero iaculis
-      interdum.
-    </p>
-    <p>Morbi vestibulum eu urna sed lacinia.</p>
   </div>
+  <ul class="dso-document-list">
+    <li>
+      <div class="dso-document-list-item">
+        <h2>Omgevingsplan gemeente Gouda</h2>
+        <div class="dso-document-list-item-container">
+          <p class="dso-document-list-item-type">Omgevingsplan</p>
+          <p class="dso-document-list-item-owner">Gemeente Gouda</p>
+          <p class="dso-document-list-item-status">
+            ${badgeTemplate({ status: 'warning', message: 'Ontwerp' })}Bekendgemaakt 01-03-2021
+          </p>
+          ${anchorTemplate({
```

Comment:
Wat gebeurt er eigenlijk als je kiest voor `Hele document bekijken` -> klapt er dan iets open of zo?
- Zou dat niet in beeld moeten komen?
- Moet dan deze link niet `aria-expanded="false"` hebben?

Reply:
Precies dat :)